indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -80,37 +89,44 @@ def _get_visible_column_ids(items):
     return (items - special_cols), (items & special_cols)
 
 
+def _get_reg_list_config(regform, allow_to_override=False):
```

Comment:
I don't think the second argument is necessary. The pages where it's set to false won't ever have a config token set and if someone provided one it wouldn't do any harm.